### DISCUSSION TITLE
What is Campus Experts?

### DISCUSSION THREAD

**Discussion by MathieuSoysal** [community/community#110934] [GitHub Education] [answered]
### Hi everyone,

Thank you for everything! GitHub Student pack is awesome. 

I'm wondered what is extacly GitHub Campus Expert ?
What is the goal ?
And how can we apply to it ?

Thnak you for your attention !

**Comment by SebastienXia** [comment, marked as answer]
Hi, @MathieuSoysal 👋

📝 To find out everything about Campus Experts, I invite you to consult the documentation on this subject: 

https://docs.github.com/fr/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-experts
